hms-app/src/components/buttons/ParticipateButton.tsx

```diff
@@ -181,18 +185,27 @@ export default function ParticipateButton(props: IProps) {
   }, [ideaData])
 
   return (
-    <Button
-      disabled={buttonIsDisabled}
-      onClick={
-        participantCheck ? removeThisIdeaParticipant : addThisIdeaParticipant
-      }
-      style={{
-        backgroundColor: participantCheck
-          ? LEAVE_BUTTON_COLOR
-          : JOIN_BUTTON_COLOR,
-      }}
-    >
-      {participantCheck ? 'Leave Idea' : 'Join Idea'}
-    </Button>
+    <>
+      <Group>
+        <Switch
+          disabled={buttonIsDisabled}
+          checked={participantCheck}
+          onChange={
+            participantCheck
+              ? removeThisIdeaParticipant
+              : addThisIdeaParticipant
+          }
+          thumbIcon={
+            participantCheck ? (
+              <IconCheck size='0.8rem' color={JOIN_BUTTON_COLOR} stroke={3} />
```

Comment:
we could use the same "Check" and "X" compenent that is used in the notifications so we wouldnt need to load 2 libarys